script_library/run-ansible.sh

```diff
@@ -6,8 +6,8 @@ function run_ansible(){
     set -x
     # ansible-runner default locations
     if [[ -z ${ANSIBLE_RUNNER_DIR+x} ]]; then
-        echo "ANSIBLE_RUNNER_DIR env var is not set, defaulting to '~/suse-osh-deploy'"
-        export ANSIBLE_RUNNER_DIR="${HOME}/suse-osh-deploy"
+        echo "ANSIBLE_RUNNER_DIR env var is not set, defaulting to '~/suse-socok8s-deploy'"
```

Comment:
is suse and socok8s redundant? "soc" could even be short and sweet.  but again, I don't want to be too nitpicky on names, so ignore me if you want to :-).

Reply:
Agreed

Reply:
I will suggest to do this as a post-merge action.
IMHO suse prefix makes it quite obvious to users (non-developers) when looking for things and then its kind of a product qualifier.
For me, it works either way.

Reply:
Fine for me to do it in another patch.